site/content/extend/plugins/example-plugins.md

```diff
@@ -1,9 +1,29 @@
 ---
-draft: true
 title: Example Plugins
 date: 2017-10-26T17:54:54-05:00
 subsection: Plugins
 weight: 40
 ---
 
-Construction
+# Example Plugins
+
+## Server "Hello, world!"
+
+To get started extending server-side functionality with plugins, take a look at our [server "Hello, world!" tutorial](../server/hello-world).
```

Comment:
Propose referring to it as a "Hello, world!" guide, for consistency with the server intro page.

(Or using tutorial in both places, 0/5 on the choice, 5/5 on consistency)